recipes/python-ecmwf_grib/build.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+if [[ $(uname) == Darwin ]]; then
+  export LIBRARY_SEARCH_VAR=DYLD_FALLBACK_LIBRARY_PATH
+elif [[ $(uname) == Linux ]]; then
+  export LIBRARY_SEARCH_VAR=LD_LIBRARY_PATH
+fi
+
+export PYTHON="$PYTHON"
+export PYTHON_LDFLAGS="$PREFIX/lib"
+export LDFLAGS="$LDFLAGS -L$PREFIX/lib -Wl,-rpath,$PREFIX/lib"
+export CFLAGS="$CFLAGS -fPIC -I$PREFIX/include"
+
+./configure --prefix=$PREFIX \
+            --with-jasper=$PREFIX \
+            --with-netcdf=$PREFIX \
+            --with-png-support \
+            --disable-fortran \
+            --enable-python
```

Comment:
I was not smart enough to trick the build and trigger python only. But the package tarball is only the python files because conda does not overwrite the files from `ecmwf_grib`.